async-stripe/src/hyper/client.rs

```diff
@@ -109,7 +109,9 @@ impl Client {
         let mut last_error = StripeError::ClientError("invalid strategy".into());
 
         if let Some(key) = strategy.get_key() {
-            req_builder = req_builder.header(HeaderName::from_static("Idempotency-Key"), key);
+            const HEADER_NAME: HeaderName = HeaderName::from_static("idempotency-key");
```

Comment:
Wow, that's a bad panic! Think this must've happened because the old usage of `http-types` instead of `hyper/http` supported this, and there must be no test coverage of idempotency key usage